.env

```diff
@@ -3,9 +3,9 @@
 REACT_APP_API_URL=http://localhost:4000
 REACT_APP_JORE_API_URL=https://dev.kartat.hsl.fi/jore/graphql
 REACT_APP_CLIENT_ID=7833861618225795
-REACT_APP_CLIENT_SECRET=
```

Comment:
Why these area removed only from .env file? Should these changes be propagated to .dev, .stage & .prod env-files?

Reply:
Removed from all envs